app/views/firms/partials/_firm_details.html.erb

```diff
@@ -96,3 +96,17 @@
     </div>
   </div>
 </div>
+
+<div class="l-firm__row">
```

Comment:
Aha ok, that's good to know. I wasn't sure if it was wrong to associate a visual style like a divider with an `l-*`, but it's ok if it's done as a modifier then?

Anyway I've remembered why I ended up doing this: it's because the `l-firm_row` is wider than the divider should be due to the way it collapses the gutters:

<img width="394" alt="screen shot 2015-10-23 at 11 02 12" src="https://cloud.githubusercontent.com/assets/306583/10689802/b05b43bc-7975-11e5-9ba7-d0df1f971924.png">

(^^ The red border is on the `l-firm__row`, the grey is the border on the column)

I definitely see some kind of `*--with-divider` is what we need but not sure what block/element to hang it off of (shrug).